### PR TITLE
Update infra README and deps

### PR DESCRIPTION
**Pull request by davidpmccormick**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # Deploy
 
 To deploy you will have to have the [AWS CLI installed](http://docs.aws.amazon.com/cli/latest/userguide/installing.html#install-with-pip)
-and your [AWS credentials setup](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html). 
+and your [AWS credentials setup](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+You will also need to have [terraform](https://www.terraform.io/downloads.html) installed.
```
Comment (jamesgorrie) [inline infra/README.md]: I preferred the update [here](https://github.com/wellcometrust/wellcomecollection.org/pull/40/files) as bullet points are easier to follow. What do you think?

**Comment by jamesgorrie** [comment]
I've updated it with a few points from mine, it's created conflicts here.
Did you get it up and going?

File: README.md [hunk 2]
```diff
@@ -2,17 +2,22 @@
...
-       
-# server
-    
-    cd server
-    npm install
-    npm run watch
-    
+## client
+
+    - `cd client`
```
Comment (jamesgorrie) [inline README.md]: Having these dashes makes it hard to copy and paste the block

File: infra/README.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 
+- [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/installing.html#install-with-pip)
+- [AWS credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html)
+- [Terraform](https://www.terraform.io/intro/getting-started/install.html)
 
-## Setup
+## Setup
 
-* You'll need to get some `tfvars` from a friendly developer near you.
+- You'll need to get some `tfvars` from a friendly developer near you.
```
Comment (jamesgorrie) [inline infra/README.md]: Any reason for these over `*` bar style?
I don't mind, just wondering.

Comment (davidpmccormick) [inline infra/README.md]: Just for consistency with the top-level README

File: README.md (hunk 2 above)
Comment (jamesgorrie) [inline README.md]: I am talking about the `-` not the `.
Also in Markdown, and indented block like that is a codeblock.

To be more explicit you could have:
<pre>
```bash
# bashcode
```
</pre>

Comment (davidpmccormick) [inline README.md]: You'd still run into trouble once you get to step three: - `npm run compile` OR `npm run compile:watch` (for developing).

I think I favour a list of things being marked up as a list. But I don't care too much.

Comment (jamesgorrie) [inline README.md]: Agreed about lists, but these are code blocks, they don't word independently of each other.
Having them as a code block allows another dev to just copy paste once.

You could have:
<pre>
```bash
cd client
npm install
npm run compile:watch # for local development
# or
npm compile # for once of compile, used on build
```
<pre>

That way you get the benefits of both?

Comment (jamesgorrie) [inline README.md]: I would however suggest for the others that have no options, that they are codeblocks and shouldn't be lists.

Comment (davidpmccormick) [inline README.md]: Right-oh – makes sense, aye.

**Pull request merged**